backend/src/routes/v2/entities/getCurrentUser.ts

```diff
@@ -34,7 +36,6 @@ export const getCurrentUser = [
   bodyParser.json(),
   async (req: Request, res: Response<GetCurrentUserResponses>) => {
     const _ = parse(req, getCurrentUserSchema)
-
-    return res.json({ user: req.user })
+    return res.json({ user: { ...req.user, isAdmin: await authentication.hasRole(req.user, Roles.Admin) } })
```

Comment:
```typescript
const _ = parse(req, getCurrentUserSchema)
const isAdmin = await authentication.hasRole(req.user, Roles.Admin)
return res.json({ user: { ...req.user, isAdmin }})
```

Splitting it out onto a separate line improves the readability, especially if we add more values.